### ISSUE TITLE
Eventwar /surrender problems

### ISSUE THREAD

**Issue by Wextra** [TownyAdvanced/Towny#6835]
### What steps will reproduce the problem?

1. War leader writes /surrender
2. The war supposedly ends
3. The war doesn't actually end

### What is the expected output?

The expected output is for the enemy war leader to get a proposal to accept or reject. In this case the enemy war leader did not even get a proposal to accept or reject, and yet according to the webhook the war ended. The war also isn't supposed to end just from a surrender offer being sent, it must be accepted too.

![image](https://github.com/TownyAdvanced/Towny/assets/35235942/1d734e6b-e1ee-4976-8f30-3dc97010a159)

In the picture, you can see in the top that the war supposedly ended with the surrender despite it not being accepted or even offered to the enemy war leader. The bottom one is when I manually ended the war.

### Towny version

Towny 0.99.5.0, Eventwar 0.0.89

### Server version

1.18.2

### Please use Pastebin.com to link the following files

1. Your full server startup from the logs\latest.log: https://pastebin.com/3d0rGe3e
2. Your Towny config.yml: https://pastebin.com/2yC7aBsQ
3. Your townyperms.yml: https://pastebin.com/FBAMahyC
4. Your log's error: -

**Comment by LlmDl** [comment]
Surrendering can be very different based on what type of war is being surrendered. For most war types surrendering doesn't require acceptance.

**Comment by LlmDl** [comment]
that does seem like a bug, i will retest it

**Comment by LlmDl** [comment]
Took a look at the code and in a nation vs nation war, when they use only /surrender, they should just be shown the help screen. What were they typing exactly when they surrendered.

**Comment by LlmDl** [comment]
OK, I took at a look at how the Surrender event is being handled in the discord embeds, and that embed text is not accurate.

It should be saying 'X is offering to surrender.' I have fixed that https://github.com/TownyAdvanced/EventWar/commit/af3ff903e90a26a736037c22b97cc7788eee673c

I believe this whole thing was just a misunderstanding because of the discord message. The war was not supposed to have ended.
